File: Python/List-1/common_end.py
# problem: https://codingbat.com/prob/p147755
# Given 2 arrays of ints, a and b, 
# return True if they have the same first element or last element.

def common_end(a, b):
  return a[0] == b[0] or a[-1] == b[-1]

File: Java/Array-1/commonEnd.java
/*
 * problem: https://codingbat.com/prob/p191991
 * Given 2 arrays of ints, a and b,
 * return true if they have the same first element or they have the same last element.
 */

public boolean commonEnd(int[] a, int[] b) {
  return (a[0] == b[0] || a[a.length - 1] == b[b.length - 1]);
}
